fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -33,13 +34,21 @@ def default_objective(self):
         """Return the default objective for moments of this kind."""
         return ErrorRate()
 
-    def load_data(self, X, y, event=None, **kwargs):
+    def load_data(self, X, y, event=None, filter_value=None, **kwargs):
         """Load the specified data into this object.
 
         This adds a column `event` to the `tags` field.
         """
         super().load_data(X, y, **kwargs)
         self.tags[_EVENT] = event
+        if filter_value is not None:
+            self.dropped_tags = self.tags.where(self.tags['label'] == 0).dropna()
+            self.tags = self.tags.where(self.tags['label'] == 1).dropna()
```

Comment:
Should these two equalities be checking against `filter_value` ?

Reply:
Yes, that is correct. I will make that change too